src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/service/JiraBuildInfoSenderImpl.java

```diff
@@ -149,7 +149,7 @@ public JiraSendInfoResponse sendBuildInfoToJiraSite(
 
     private Optional<String> getAccessTokenFor(
             final JiraCloudSiteConfig siteConfig, final String secret) {
-        final AppCredential appCredential = new AppCredential(siteConfig.getClientId(), secret);
+        final AppCredential appCredential = new AppCredential(siteConfig.getWebhookUrl(), secret);
```

Comment:
I just renamed the field `clientId` to `webhookUrl`, but it's still being interpreted as a clientId in much of the backend. Following up on that in the next PR.